vita3k/ngs/src/modules/player.cpp

```diff
@@ -53,14 +53,14 @@ void PlayerModule::on_param_change(const MemState &mem, ModuleData &data) {
 
     // check for invalid playback values
     const auto is_invalid_playback_value = [](const float playback_value, const float max_value) {
-        return isnan(playback_value) || (playback_value < 0.f) || (playback_value > max_value);
+        return isnan(playback_value) || (playback_value <= 0.f) || (playback_value > max_value);
```

Comment:
?? 0 value is totaly valid for ngs

Reply:
ah after test it i understund issue... 
relate scaling, it try re scaling with 0 value, and like unity game, it do'nt like that